mage_ai/server/api/logs.py

```diff
@@ -60,7 +61,8 @@ def get(self, pipeline_uuid):
             PipelineRun.
             select(*columns).
             join(b, a.pipeline_schedule_id == b.id).
-            filter(b.pipeline_uuid == pipeline_uuid)
+            filter(b.pipeline_uuid == pipeline_uuid).
+            order_by(a.execution_date.desc())
```

Comment:
@wangxiaoyou1993 If I want to fetch the latest logs first, is this the right way to order them? It's so I can set the `_offset` and `_limit` initially at 0 and 20, respectively, and then increase them in intervals. But there seems to be some mix of logs not chronologically dated correctly when fetching log files.

For example in the summary gif, the oldest log in the first request is from 11/1/22, but when fetching the next (older) interval of logs, it includes logs from 11/9/22 as well.